Utilities/aliceHLTwrapper/src/MessageFormat.cxx

```diff
@@ -134,7 +134,7 @@ int MessageFormat::addMessage(uint8_t* buffer, unsigned size)
   return mBlockDescriptors.size() - count;
 }
 
-int MessageFormat::addMessages(const vector<BufferDesc_t>& list)
```

Comment:
These changes (adding a `std::`) can also go in as soon as possible. Could you factor them out in a separate PR?